packages/components/src/Modal.test.tsx

```diff
@@ -12,14 +11,9 @@ describe('Modal', () => {
   const renderModal = (props: Partial<ModalProps> = {}): void => {
     const { children, ...otherProps } = props;
     renderWithTheme(
-      h(
-        Modal,
-        {
-          onDismiss: jest.fn(),
-          ...otherProps,
-        },
-        [children]
-      )
+      <Modal onDismiss={jest.fn()} {...otherProps}>
```

Comment:
In .eslintrc.js, I saw `react/jsx-props-no-spreading': 'off'`
Wouldn't this prevent `{...otherProps}`, or am I misunderstanding what that property does?

Reply:
That rule disallows `{...otherProps}`. Turning the rule off permits it.

https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-props-no-spreading.md

Reply:
Ahh ok, that makes sense. Thanks for clarifying.